.github/workflows/CI.yml

```diff
@@ -56,20 +56,4 @@ jobs:
             openbox &
             cargo test --manifest-path relm-examples/Cargo.toml --examples
 
-      - name: "relm: test buttons-attribute example"
-        run: |
-            Xvfb :99 &
-            sleep 3
-            openbox &
-            cargo test --manifest-path relm-examples/examples/buttons-attribute/Cargo.toml
-
-      - name: "relm: build http example"
-        run: cargo build --manifest-path relm-examples/examples/http/Cargo.toml
-
-      - name: "relm: build async example"
-        run: cargo build --manifest-path relm-examples/examples/async/Cargo.toml
-
-      - name: "relm: build webkit-test example"
```

Comment:
The webkit example was kept separate so that if it fails because the crate is outdated or something, it's easy to see.

Reply:
Got it, I can move it back to its own crate. Would you like me to keep the other examples separate as well? The `buttons-attribute` test appeared to be duplicated.